src/components/CourseChecklist/index.jsx

```diff
@@ -119,6 +120,13 @@ class CourseChecklist extends React.Component {
         className={classNames('px-3', styles.btn, styles['btn-primary'], styles['checklist-item-link'])}
         content={<WrappedMessage message={messages.updateLinkLabel} />}
         destination={this.getUpdateLinkDestination(checkID)}
+        onClick={() => trackEvent(
```

Comment:
To make sure that I understand > it would still be an anon function, but have it call the separate functions noted in your comment to handle the extra parameters passed through for the `trackEvent` call, right?

Reply:
I'll push up what I'm thinking, and you can left me know if you disagree.